inc/class-registration.php

```diff
@@ -590,6 +590,11 @@ public function enqueue_block_styles( $post ) {
 
 			array_push( self::$styles_loaded, $block );
 		}
+
+		if ( $otter_block_found ) {
```

Comment:
None of the CSS in the file is related to any of these blocks, so this doesn't do anything for us. 😄